src/Model/CategoryManager.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Model;
+
+use PDO;
+
+class CategoryManager extends AbstractManager
+{
+    private const TABLE_CATEGORY = 'category';
+
+    public function getAllCategories(): array
+    {
+        $statement = $this->pdo->query("SELECT name FROM " . self::TABLE_CATEGORY);
+
+        return $statement->fetchAll(PDO::FETCH_ASSOC);
+    }
```

Comment:
Not necessary, with AbstractManager as a Parent, you already have a selectAll() in the Manager with the same job